packages/react-native-bpk-component-button/src/__snapshots__/BpkButton-test.android.js.snap

```diff
@@ -193,7 +193,7 @@ exports[`Android BpkButton dark mode should render correctly 1`] = `
     nativeBackgroundAndroid={
       Object {
         "borderless": true,
-        "color": null,
+        "color": -9934465,
```

Comment:
It's because of this logic https://github.com/facebook/react-native/blob/505a2167f50213c4d237b387fba41de215a6140c/Libraries/StyleSheet/processColor.js#L18.

Basically RN transforming the string color into something Android understands